docs/developer/api-reference/enums/order-create-from-checkout-error-code.mdx

```diff
@@ -0,0 +1,55 @@
+---
+id: order-create-from-checkout-error-code
+title: OrderCreateFromCheckoutErrorCode
+---
+
+An enumeration.
+
+```graphql
+enum OrderCreateFromCheckoutErrorCode {
+  GRAPHQL_ERROR
+  NOT_FOUND
```

Comment:
Can we make it a little bit more verbose?
```suggestion
  CHECKOUT_NOT_FOUND
```

Reply:
I can, but you will also get this error code attached to a field. For `OrderCreateFromCheckoutErrorCode` you will get an errors like {"field": "id", "code": "NOT_FOUND", "message": "Couldn't resolve to a node: <ID>"}` 
If it is not enough context I can create a PR to update this and conver NOT_FOUND to CHECKOUT_NOT_FOUND

Reply:
I'm forwarding Saleor error codes from this mutation back to the client when something fails, a `NOT_FOUND` error is not saying much if you're treating payments app as a blackbox. So being more explicit makes sense to me

Reply:
Okay, sounds good. I will provide a PR to change this. Thanks for raising this